workspaces/ui-v2/src/optic-components/hooks/useShapeDescriptor.ts

```diff
@@ -49,6 +54,30 @@ const changesSinceShapeQuery = `
     }
 }`;
 
+type ShapeChoice =
```

Comment:
I'd want to see if there's a way to generate typings off gql queries, but I'm not sure how practical that is - in the meantime, this needs to be kept in sync with the gql query above